apiv2/db_import/importers/annotation.py

```diff
@@ -0,0 +1,237 @@
+        return data_map
+
+    @classmethod
+    def get_id_fields(cls) -> list[str]:
+        return ["format", "tomogram_voxel_spacing_id", "annotation_shape_id"]
+
+    @classmethod
+    def get_db_model_class(cls) -> type[Base]:
+        return models.AnnotationFile
+
+    def get_filters(self) -> dict[str, Any]:
+        return {"tomogram_voxel_spacing_id": self.tomogram_voxel_spacing_id}
+
+    @classmethod
+    def get_item(
+        cls,
+        annotation_shape_id: int,
+        tomogram_voxel_spacing_id: int,
+        parent: AnnotationDBImporter,
+        config: DBImportConfig,
+    ) -> "AnnotationFilesDBImporter":
+        return cls(annotation_shape_id, tomogram_voxel_spacing_id, parent, config)
+
+
+class AnnotationShapesDBImporter(StaleDeletionDBImporter):
+    def __init__(self, annotation_id: int, parent: AnnotationDBImporter, config: DBImportConfig):
+        self.annotation_id = annotation_id
+        self.parent = parent
+        self.config = config
+        self.metadata = parent.metadata.get("files", [])
+
+    def get_data_map(self) -> dict[str, Any]:
+        return {
+            "annotation_id": self.annotation_id,
+            "shape_type": ["shape"],
+        }
+
+    @classmethod
+    def get_id_fields(cls) -> list[str]:
+        return ["annotation_id", "shape_type"]
+
+    @classmethod
+    def get_db_model_class(cls) -> type[Base]:
+        return models.AnnotationShape
+
+    def get_filters(self) -> dict[str, Any]:
+        return {"annotation_id": self.annotation_id}
+
+    @classmethod
+    def get_item(
+        cls,
+        annotation_id: int,
+        parent: AnnotationDBImporter,
+        config: DBImportConfig,
+    ) -> "AnnotationShapesDBImporter":
+        return cls(annotation_id, parent, config)
+
+
+class AnnotationAuthorDBImporter(AuthorsStaleDeletionDBImporter):
+    def __init__(self, annotation_id: int, parent: AnnotationDBImporter, config: DBImportConfig):
+        self.annotation_id = annotation_id
+        self.parent = parent
+        self.config = config
+        self.metadata = parent.metadata.get("authors", [])
+
+    def get_data_map(self) -> dict[str, Any]:
+        return {
+            "annotation_id": self.annotation_id,
+            "orcid": ["ORCID"],
+            "name": ["name"],
+            "corresponding_author_status": ["corresponding_author_status"],
+            "email": ["email"],
+            "affiliation_name": ["affiliation_name"],
+            "affiliation_address": ["affiliation_address"],
+            "affiliation_identifier": ["affiliation_identifier"],
+            "author_list_order": ["author_list_order"],
+        }
+
+    def update_data_map(self, data_map: dict[str, Any], metadata: dict[str, Any], index: int) -> dict[str, Any]:
+        data_map = super().update_data_map(data_map, metadata, index)
+        primary_author_status = {
+            "primary_annotator_status": metadata.get("primary_annotator_status", metadata.get("primary_author_status")),
```

Comment:
Could we use this migration as an opportunity to deprecate `primary_annotator_status` in favour of `primary_author_status`?